cli/server.go

```diff
@@ -739,6 +739,10 @@ func (r *RootCmd) Server(newAPI func(context.Context, *coderd.Options) (*coderd.
 					_ = sqlDB.Close()
 				}()
 
+				if options.DeploymentValues.Prometheus.Enable {
+					options.PrometheusRegistry.MustRegister(collectors.NewDBStatsCollector(sqlDB, ""))
```

Comment:
I'd suggest adding a comment with our rationale for setting dbName to an empty string.